db/migrate/20190121210000_create_recipes.rb

```diff
@@ -0,0 +1,18 @@
+# Create Recipes Table
+
+class CreateRecipes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :recipes do |t|
+      ## 
+      t.string :title,       null: false, default: ""
+      t.string :snipped,     null: false, default: ""
+      t.string :meal,        null: false, default: ""
+      t.text   :ingredients, array: true, default: []
+      t.text   :directions,  array: true, default: []
+      t.string :difficulty,  null: false, default: "Intermediate"
+      t.integer :duration,   null: false, default: 0
+      t.integer :servings,   null: false, default: 2
+      t.string  :photos,     array: true, default: []    
```

Comment:
I think it'd be good to add `t.timestamps` at the end here, which will add created_at, updated_at, and deleted_at timestamps.